common4j/src/main/com/microsoft/identity/common/java/AuthenticationConstants.java

```diff
@@ -588,6 +588,11 @@ public static final class OAuth2Scopes {
          */
         public static final String EMAIL_SCOPE = "email";
 
+        /**
+         * Scope to get transfer token
+         */
+        public static final String TRANSFER_TOKEN_SCOPE = "service::http://Passport.NET/purpose::PURPOSE_TRANSFER_TOKEN_BACKUP_RESTORE";
```

Comment:
What is Passport.NET?

Reply:
Not sure what Passport.NET signifies. But this is the custom scope required for getting transfer token instead of an access token using PRT. As per MSA server side spec https://msazure.visualstudio.com/One/_git/ESTS-Docs/pullrequest/7258879?_a=files&path=/TransferToken/BackupAndRestore.md